roles/tpa_single_node/vars/main.yml

```diff
@@ -8,38 +8,38 @@ tpa_single_node_namespace: trustification
 tpa_single_node_podman_network: tcnet
 tpa_single_node_skip_os_install: true
 tpa_single_node_systemd_directory: /etc/systemd/system
+tpa_single_node_default_empty: ""
 
 # DB_SERVICE
-# TODO - Workaround for PostgreSQL PG_HBA issue
 tpa_single_node_pg_host: 192.168.121.60
 tpa_single_node_pg_port: 5432
 tpa_single_node_pg_db: guac
 tpa_single_node_pg_user: guac
 tpa_single_node_pg_user_passwd: guac1234
 tpa_single_node_pg_ssl_mode: disable
 
-# S3_SERVICE
-tpa_single_node_s3_url:
-tpa_single_node_s3_access_key: "{{ lookup('env', 'TPA_S3_ACCESS_KEY') }}" # Export S3 access key
-tpa_single_node_s3_secret_key: "{{ lookup('env', 'TPA_S3_SECRET_KEY') }}" # Export S3 Secret key
-tpa_single_node_s3_v11y_bucket: # <v11y storage bucket name>
-tpa_single_node_s3_storage_region: # <AWS S3 Stroage region> # For Minio just keep us-west-1
-tpa_single_node_s3_minio_endpoint: # <Enter minio storage endpoint, For AWS this field is not necessary>
+# Storage Service (S3 or Minio)
+tpa_single_node_storage_endpoint: # <Enter minio storage endpoint, For AWS this field is not necessary>
+tpa_single_node_storage_region: # <AWS S3 Storage region> # For Minio just keep us-west-1
+tpa_single_node_storage_access_key_id: "{{ lookup('env', 'TPA_STORAGE_ACCESS_KEY_ID') }}"
+tpa_single_node_storage_secret_access_key: "{{ lookup('env', 'TPA_STORAGE_SECRET_ACCESS_KEY') }}"
+tpa_single_node_storage_bombastic_bucket: bombastic-default # <bombastic storage bucket name>
+tpa_single_node_storage_v11y_bucket: v11y-default # <v11y storage bucket name>
 
 # SQS_SERVICE
 
-# SSO_SERVICE
-tpa_single_node_oidc_issuer_url: # <Keycloak or AWS cognito Issuer URL with endpoint auth/realms/chicken>
-tpa_single_node_oidc_frontend: # <Keycloak or AWS cognito frontend Client ID>
-tpa_single_node_oidc_walker: # <Keycloak or AWS cognito Walker Client ID>
-tpa_single_node_oidc_walker_secret: "{{ lookup('env', 'TPA_OIDC_WALKER_SECRET') }}" #  Export AWS Cognito or Keycloak walker Secret
+# SSO_SERVICE - Keycloak or AWS Cognito
+# Issuer URL with endpoint auth/realms/chicken
+tpa_single_node_oidc_issuer_url: "{{ lookup('env', 'TPA_STORAGE_OIDC_ISSUER_URL') }} | default('tpa_single_node_default_empty', true)"
```

Comment:
This conditio not working properly
It could be 
`tpa_single_node_oidc_issuer_url: "{{ lookup('env', 'TPA_STORAGE_OIDC_ISSUER_URL') | default('tpa_single_node_default_empty', true) }}"`
Environment value
![image](https://github.com/user-attachments/assets/5c783098-ebf4-476f-bef7-c4baccc22a9f)
Value on yaml file
![image](https://github.com/user-attachments/assets/0219f9ee-4c03-4f60-b404-d2ac7a081fb4)
Error from bombastic-api
![image](https://github.com/user-attachments/assets/619cdcd4-b873-4bdb-81f9-be4f9d713830)